Build error by Carthage

I tried building by Carthage and received the following error.


```
Undefined symbols for architecture x86_64:
  "direct type metadata for ext.Carlos.__ObjC.NSDateFormatter.Error", referenced from:
      protocol witness for Swift.ErrorType._domain.getter : Swift.String in conformance ext.Carlos.__ObjC.NSDateFormatter.Error : Swift.ErrorType in Carlos in ConcurrentOperation.o
  "direct type metadata for ext.Carlos.__ObjC.NSNumberFormatter.Error", referenced from:
      protocol witness for Swift.ErrorType._domain.getter : Swift.String in conformance ext.Carlos.__ObjC.NSNumberFormatter.Error : Swift.ErrorType in Carlos in ConcurrentOperation.o
ld: symbol(s) not found for architecture x86_64
```

By CocoaPods, Building succeeded.

--- a/Carlos.xcodeproj/project.pbxproj
+++ b/Carlos.xcodeproj/project.pbxproj
@@ -1919,7 +1919,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "de.weltn24.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
-				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				SWIFT_OPTIMIZATION_LEVEL = "-O";
 				VERSIONING_SYSTEM = "apple-generic";
 				VERSION_INFO_PREFIX = "";
 			};

